ci/logstash_releases.json

```
@@ -1,12 +1,12 @@
 {
   "releases": {
     "7.current": "7.17.28",
-    "8.previous": "8.17.4",
+    "8.previous": "8.17.5",
     "8.current": "8.18.0"
   },
   "snapshots": {
     "7.current": "7.17.29-SNAPSHOT",
-    "8.previous": "8.17.5-SNAPSHOT",
+    "8.previous": "8.17.6-SNAPSHOT",
     "8.current": "8.18.1-SNAPSHOT",
     "8.next": "8.19.0-SNAPSHOT",
     "9.next": "9.0.0-SNAPSHOT",
```

Can we also include 9.0 change?
```suggestion
    "9.next": "9.0.1-SNAPSHOT",
```